Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableShaderTheme.cs

```diff
@@ -22,14 +23,16 @@ public class InteractableShaderTheme : InteractableThemeBase
         protected MaterialPropertyBlock propertyBlock;
         protected List<ThemeStateProperty> shaderProperties;
         protected Renderer renderer;
+        private Graphic graphic;
+        private Material material;
```

Comment:
>private Material material; [](start = 8, length = 26)

I don't think you should store this material.  You can just access it from the graphic every time.